integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -85,9 +88,42 @@ public void userJourney(TestUserSpecification testUser) throws Exception {
     datasetsApi = new DatasetsApi(client);
 
     crudUserJourney(client, StorageSystem.TDR, snapshotId.toString());
-    crudUserJourney(client, StorageSystem.WKS, workspaceDetails.getWorkspaceId());
+    crudUserJourney(client, StorageSystem.WKS, workspaceSource.getWorkspaceId());
 
     previewUserJourney(StorageSystem.TDR, snapshotId.toString());
+
+    exportUserJourney(StorageSystem.WKS, workspaceSource);
+  }
+
+  private void exportUserJourney(StorageSystem storageSystem, WorkspaceDetails workspaceSource)
+      throws ApiException, bio.terra.rawls.client.ApiException {
+
+    // Create workspace dataset
+    var request =
+        new CreateDatasetRequest()
+            .catalogEntry(METADATA_1)
+            .storageSourceId(workspaceSource.getWorkspaceId())
+            .storageSystem(storageSystem);
+    datasetId = datasetsApi.createDataset(request).getId();
+    workspaceDest = rawlsClient.createTestWorkspace();
+
+    // Export workspace to workspace
+    var workspaceId = UUID.fromString(workspaceDest.getWorkspaceId());
+    datasetsApi.exportDataset(datasetId, workspaceId);
+
+    // Extract workspace entity names
+    Set<String> entitiesSource = rawlsClient.getWorkspaceEntities(workspaceSource);
+    Set<String> entitiesDest = rawlsClient.getWorkspaceEntities(workspaceDest);
+
+    assertEquals(entitiesSource, entitiesDest);
```

Comment:
What entities are in these sets? Right now I don't think `workspaceSource` contains any entities. Do you plan to wait for Omesi's PR to merge up, which does populate the workspace?